tasks/copy_to_glacier/schemas/input.json

```diff
@@ -0,0 +1,47 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://github.com/nasa/cumulus-orca/blob/master/tasks/copy_to_files/schemas/input.json",
+  "title": "copy_to_files Lambda Input",
+  "description": "The input for the copy_to_files Lambda.",
+  "type": "object",
+  "required": ["granules"],
+  "properties": {
+    "granules": {
+      "description": "A list of objects representing individual files to copy.",
+      "type": "array",
+      "items": {
+        "required": ["granuleId", "files"],
+        "properties": {
+          "granuleId": {
+            "description": "The granule's ID.",
+            "type": "string"
+          },
+          "files": {
+            "description": "The files to copy.",
+            "type": "array",
+            "items": {
+              "required": ["name", "bucket", "filepath", "filename"],
+              "properties": {
+                "name": {
+                  "type": "string"
```

Comment:
should add a description to each of these properties so users know what to expect for values.

Reply:
Gave it a shot.